### DISCUSSION TITLE
CD-RW & DVD-R colored icons.

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#14] [Ideas]
What do you think about idea of CD-RW & DVD-R colored icons?

![cd-rw_13_256x256x8](https://user-images.githubusercontent.com/26491673/205482123-1c28f039-09ec-488a-8351-27063fbc3ca4.png) ![dvd-r_13_256x256x8](https://user-images.githubusercontent.com/26491673/205482107-8f01e487-fb42-4bb2-a7c7-4107b6748d7e.png)

**Comment by Dejweed** [comment]
Interesting idea. Can you send a picture of how it would look like in action?
